<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SprintValidationPage</class>
 <widget class="QWidget" name="SprintValidationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Sprint Pre-Execution Check</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>The system has analyzed the selected sprint items. Please review the report below before proceeding.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="reportStackedWidget">
     <widget class="QWidget" name="processingPage">
      <layout class="QVBoxLayout" name="verticalLayout_proc">
       <item>
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="processingLabel">
         <property name="text">
          <string>Running validation checks...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="reportPage">
      <layout class="QVBoxLayout" name="verticalLayout_report">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>742</width>
            <height>405</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="scrollAreaLayout">
           <item>
            <widget class="QLabel" name="scopeHeaderLabel">
             <property name="text">
              <string>Scope Guardrail Check Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="scopeDetailsTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="staleHeaderLabel">
             <property name="text">
              <string>Stale Analysis Check Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="staleDetailsTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="riskHeaderLabel">
             <property name="text">
              <string>Technical Risk Check Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="riskDetailsTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="returnToBacklogButton">
       <property name="text">
        <string>Return to Backlog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="proceedButton">
       <property name="text">
        <string>Proceed to Plan Generation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>